app/Http/routes.php

```diff
@@ -31,9 +31,13 @@
         return 'Rogue API version 1';
     });
 
-    Route::get('reportbacks', 'Api\ReportbackController@index');
-    Route::post('reportbacks', 'Api\ReportbackController@store');
+    // /items
     Route::put('items', 'Api\ReportbackController@updateReportbackItems');
+
+    // /reactions
     Route::post('reactions', 'Api\ReactionController@store');
-    Route::get('users', 'UsersController@index');
```

Comment:
@chloealee I think it's fine as long as it doesn't affect anything like logging in! I can't think of any reason to keep it there if it's not used